### PR TITLE
smiley-sans: init at 1.0.0

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/smiley-sans/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  version = "1.0.0";
+
+  src = fetchurl {
+    url = "https://github.com/atelier-anchor/smiley-sans/releases/download/v${version}/smiley-sans-v${version}.zip";
+    sha256 = "sha256-gpPJuf1Eye5rP6tpaGJEUFnk2Ys0GhSNRUT5HQE2P8E=";
+  };
+
+  unpackPhase = ''
+    ${unzip}/bin/unzip $src
+  '';
```
Comment (figsoda) [inline pkgs/data/fonts/smiley-sans/default.nix:14]:
```suggestion
  src = fetchzip {
    url = "https://github.com/atelier-anchor/smiley-sans/releases/download/v${version}/smiley-sans-v${version}.zip";
    sha256 = "sha256-LE0CZkWiXjyuiEk316ABCNQL9n5GID8CipjBIu2o6uk=";
    stripRoot = false;
  };
```

File: pkgs/data/fonts/smiley-sans/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/fonts
```
Comment (figsoda) [inline pkgs/data/fonts/smiley-sans/default.nix:18]: `install` already does that
```suggestion
```

File: pkgs/data/fonts/smiley-sans/default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    install -Dm644 -t $out/share/fonts/opentype *.otf
+    install -Dm644 -t $out/share/fonts/truetype *.ttf
+    install -Dm644 -t $out/share/fonts/smiley-sans *.woff2
```
Comment (figsoda) [inline pkgs/data/fonts/smiley-sans/default.nix:21]:
```suggestion
    install -Dm644 -t $out/share/fonts/woff2 *.woff2
```

**Review by figsoda — CHANGES_REQUESTED** [review]

**Review by figsoda — APPROVED** [review]

**Comment by candyc1oud** [comment]
@figsoda I fixed, thanks

**Pull request merged**